src/Options.js

```diff
@@ -164,7 +165,7 @@ export class Options {
   }
 
   get sourceMaps() {
-    return this.sourceMaps_;
+    return this.lowResolutionSourceMap || this.sourceMaps_;
```

Comment:
If `this.lowResolutionSourceMap` is true, then we will return `true` but that is not valid value for this property correct?